common/src/main/java/org/opensearch/ml/common/parameter/FunctionName.java

```diff
@@ -8,7 +8,7 @@
 public enum FunctionName {
     LINEAR_REGRESSION,
     KMEANS,
-    ANOMALY_DETECTION,
+    AD_LIBSVM,
```

Comment:
I may lack the context - what is the difference between these two names? They represent the same AD algorithm?

Reply:
That's to avoid confusion. Tribuo using libSVM to do anomaly detection. The new name is more accurate. So user can distinguish different AD algorithms.